src/GeneralStateTestsFiller/stCreate2/CREATE2_FirstByte_loopFiller.yml

```diff
@@ -0,0 +1,59 @@
+# The test calls CREATE2 in a loop deploying 1-byte contracts with all possible byte values, records in storage the values that failed to deploy.
+CREATE2_FirstByte_loop:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 100000000
+      code: ''
+      nonce: 0
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul
+        {
+          // { mstore8(0, 0x00) return(0, 1) }
+          mstore(0, 0x600060005360016000f300000000000000000000000000000000000000000000)
+          for { let code := 0 } lt(code, 256) { code := add(code, 1) } 
+          { 
+            mstore8(1, code)
+            if iszero(create2(0, 0, 10, 0)) { sstore(code, 1) }
+          }
+          sstore(256, 1)
+        }
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data: 
+      - ''
+    gasLimit:
+      - 70000000
+    gasPrice: 1
+    nonce: 0
+    secretKey: 45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8
+    to: 'b94f5374fce5edbc8e2a8697c15331677e6ebf0b'
+    value:
+    - 0
+
+  expect:
+    - indexes:
+       data: !!int -1
+       gas: !!int -1
+       value: !!int -1
+      network:
+        - 'Berlin'
+      result:
+       a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
```

Comment:
can add a few addresses with deployed code here to show and to see that accounts with that bytecode has actually been deployed

Reply:
Added